src/core/cmap.js

```diff
@@ -996,7 +996,7 @@ var CMapFactory = (function CMapFactoryClosure() {
   }
 
   return {
-    create(params) {
```

Comment:
The source of the error comes from https://github.com/mozilla/pdf.js/blob/517ccb7a394148942616433d49782cc8126a49fe/src/core/cmap.js#L1008 and rather than wrapping all this method in a `Promise` it seemed easiest to just make it `async` instead.